server/index.ts

```diff
@@ -1,13 +0,0 @@
-import express, { Express, Request, Response } from 'express'
-
-const port = 8000
-const app: Express = express()
-
-app.get('/', (req: Request, res: Response) => {
-	res.send
-('Hiii!')
-})
-
-app.listen(port, () => {
-	console.log(`Server running on port ${port}`)
-})
```

Comment:
Why is this deleted btw?

Reply:
I just deleted because I re-started from another approach, I guess we can talk about this on Monday